cfgov/v1/migrations/0233_clean_orphaned_revisions.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.2.17 on 2023-03-14 13:09
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Page = apps.get_model('wagtailcore', 'Page')
+    PageRevision = apps.get_model('wagtailcore', 'PageRevision')
```

Comment:
How will this work when we're on Wagtail 4, where PageRevision is no longer a thing? Will we need to do migration surgery to this file to rename this to Revision?

Reply:
This will have to apply before Wagtail 4, basically, because Wagtail 4's migration that migrates `PageRevision` to `Revision` will fail with these orphaned revisions in the database.

Reply:
...as I just discovered when I tried to use Wagtail 4 locally. But I tried this migration and it works like a charm.